terraform/cicd-deployment/main.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_cloudformation_stack" "data_exports_child" {
+  # checkov:skip=CKV_AWS_124:SNS topic not required for this use case
+  provider     = aws.destination_account
+  name         = "data-exports-aggregation-child"
+  template_url = "https://aws-managed-cost-intelligence-dashboards.s3.amazonaws.com/cfn/${var.global_values.tag_version}/data-exports-aggregation.yaml"
```

Comment:
data-exports-aggregation has its own versioning so there no such thing as tag_version. Must be data_exports_stack_version or something of this sort